hack/jenkins/e2e-runner.sh

```diff
@@ -376,7 +376,7 @@ if [[ "${E2E_DOWN,,}" == "true" ]]; then
 fi
 
 if [[ "${E2E_TEST,,}" == "true" ]]; then
-  e2e_go_args+=(--test --test_args="${GINKGO_TEST_ARGS}")
+  e2e_go_args+=(--test --test_args="${GINKGO_TEST_ARGS:-}")
```

Comment:
maybe `${test_args:+--test_args="${test_args}"}` so we actually only set a value for `--test_args` when set? minor nit though.